### DISCUSSION TITLE
Fluent API support?

### DISCUSSION THREAD

**Discussion by alexanderfedin** [angularsen/UnitsNet#1117] [Ideas]
Did you think of adding fluent-like APIs:
var speed = 12.Meters() / 3.5.Seconds();

where:
public static class DistanceFluentDSL
{
    public static Distance Meters(this int v) => new Distance(v, Unit.Meters);
    public static Distance Meters(this double v) => new Distance(v, Unit.Meters);
}

**Comment by angularsen** [comment]
Hi, check out https://www.nuget.org/packages/UnitsNet.NumberExtensions/
